frontend/src/components/pages/admin/DisplayAssessmentsPage.tsx

```diff
@@ -193,74 +102,88 @@ const DisplayAssessmentsPage = (): React.ReactElement => {
           <CreateAssessementButton />
         </HStack>
       </Box>
-      {/* {loading && (
-        <Center margin="15%" flex="1">
+      {loading && (
+        <Center flex="1" margin="15%">
           <LoadingState />
         </Center>
       )}
       {error && (
-        <Center margin="15%" flex="1">
+        <Center flex="1" margin="15%">
           <ErrorState />
         </Center>
-      )} */}
-      {/* {sampleAssessments && !error && !loading && ( */}
-      <Box flex="1">
-        <Tabs marginTop={3}>
-          <TabList>
-            <Tab color={unselectedColor} onClick={() => setSearch("")}>
-              All
-            </Tab>
-            <Tab color={unselectedColor} onClick={() => setSearch("Draft")}>
-              Drafts
-            </Tab>
-            <Tab color={unselectedColor} onClick={() => setSearch("Published")}>
-              Published
-            </Tab>
-            <Tab color={unselectedColor} onClick={() => setSearch("Archived")}>
-              Archived
-            </Tab>
-          </TabList>
-          <TabPanels>
-            <TabPanel>
-              <VStack pt={4} spacing={6}>
-                <HStack width="100%">
-                  <SearchBar onSearch={setSearch} />
-                  <SortMenu
-                    onSortOrder={setSortOrder}
-                    onSortProperty={setSortProperty}
-                    properties={[
-                      "status",
-                      "name",
-                      "grade",
-                      "type",
-                      "country",
-                      "region",
-                    ]}
-                  />
-                  <FilterMenu />
-                </HStack>
-                {search && (
-                  <Text color="grey.300" fontSize="16px" width="100%">
-                    Showing {sampleAssessments.length} results for &quot;
-                    {search}&quot;
-                  </Text>
-                )}
+      )}
+      {data && !error && !loading && (
+        <Box flex="1">
+          <Tabs marginTop={3}>
+            <TabList>
+              <Tab color={unselectedColor} onClick={() => setSearch("")}>
+                All
+              </Tab>
+              <Tab color={unselectedColor} onClick={() => setSearch("Draft")}>
+                Drafts
+              </Tab>
+              <Tab
+                color={unselectedColor}
+                onClick={() => setSearch("Published")}
+              >
+                Published
+              </Tab>
+              <Tab
+                color={unselectedColor}
+                onClick={() => setSearch("Archived")}
+              >
+                Archived
+              </Tab>
```

Comment:
is the functionality for switching tabs implemented in a diff pr?

Reply:
Yup, in the filter menu pr!